envsec/internal/jetcloud/client.go

```diff
@@ -0,0 +1,30 @@
+package jetcloud
+
+import (
+	"fmt"
+	"os"
+
+	"go.jetpack.io/envsec/internal/auth"
+	typeid "go.jetpack.io/typeid/typed"
+)
+
+type projectPrefix struct{}
+
+func (projectPrefix) Type() string { return "project" }
```

Comment:
In the DB, are we going to have a single projects table for all of our products or is the projects table specific to envsec? If specific to envsec, maybe we want `envsec_project`? (we'll have to change typeid to allow for `_` in the prefix, but I've been wanting to do that for a while)

Reply:
Can we do `proj_` instead? Also, I would prefer that the project id format is consistent across all of our products